packages/plugins/flow/operations/src/flow-selection-set-processor.ts

```diff
@@ -73,7 +73,7 @@ export class FlowWithPickSelectionSetProcessor extends BaseSelectionSetProcessor
 
   transformPrimitiveFields(
     schemaType: GraphQLObjectType | GraphQLInterfaceType,
-    fields: PrimitiveField[]
+    fields: PrimitiveWithFlagsField[]
```

Comment:
You can just change the `PrimitiveField` type to match your needs. This way we can spot other places where this change it needed.